components/rss/app/rss.app.ts

```diff
@@ -4,6 +4,7 @@ import { Item } from "feedparser";
 import hash from "object-hash";
 import { defineApp } from "@pipedream/types";
 import { ConfigurationError } from "@pipedream/platform";
+import { ReadStream } from "fs";
 
```

Comment:
Is there any reason we can't use Pipedream axios?
`import { axios } from "@pipedream/platform";`
instead of 
`import axios from "axios";`

Reply:
I believe that it's required so it can be used in the typescript env with "@types/axios", few weeks ago I changed it to Pipedream's axios here: https://github.com/PipedreamHQ/pipedream/pull/2697/commits/6b066fef4abae5b7814f02098bb0f0baea7243f0 

But for some reason @dylburger changed to global axios after here: https://github.com/PipedreamHQ/pipedream/pull/2780/commits/8aea14a21e301025adaf6b8c8c7109e4e4eebf95